src/components/DashboardSpeakers/DashboardSpeakersPresentation.js

```diff
@@ -23,7 +24,7 @@ function DashboardSpeakersPresentation({
         site={site}
       >
         <DashboardTable.Presentation
-          queryResponse={queryResponse}
+          infiniteQueryResponse={infiniteQueryResponse}
```

Comment:
`DashboardTable` is still expecting the prop to be named `queryResponse` so you will need to pass it as such here i.e. 
`<DashboardTable.Presentation
          queryResponse={infiniteQueryResponse}`